### ISSUE TITLE
[Class modifiers] The `Function` class will be final in Dart 3.0

### ISSUE THREAD

**Issue by lrhn** [dart-lang/co19#1874]
In pre-3.0 code the `Function` class can be implemented, extended and (probably) mixed in, it's just ignored and has absolutely no effect.
In 3.0 code, the `Function` class will have a `final` modifier which prevents all of the above.

The following tests seem to be affected by that (according to a `grep`):
```
Language/Expressions/Method_Invocation/Ordinary_Invocation/accessible_instance_member_t07.dart:class A extends Function {
Language/Expressions/Method_Invocation/Ordinary_Invocation/accessible_instance_member_t08.dart:class A extends Function {
```

If there are more examples (didn't find any `implements`?) or ones that go through indirections, say something like `implements IdAlias<Function>` with `typedef IdAlias<T> = T;`, then they might also need fixing.

**Comment by sgrekhov** [comment]
@mit-mit the log is unavailable for me. Could you, please, provide a link to the gerrit CL?

**Comment by mit-mit** [comment]
Sorry about that. It's here: https://dart-review.googlesource.com/c/sdk/+/284185?tab=checks

And the test results: https://ci.chromium.org/ui/p/dart/builders/try/analyzer-linux-release-try/63852/overview

**Comment by sgrekhov** [comment]
Now I see. Thank you!

**Comment by sgrekhov** [comment]
If you don't mind I'll fix if when CL is landed

**Comment by sgrekhov** [comment]
Fixed by ba5ea0ef34d3f67b169b4b30ed2a0a5f1a354cef and cc94e132caaad644c618d13b77ec2c27e707d263
